src/settings.h

```diff
@@ -12,6 +12,7 @@ enum icon_position_t { icons_left, icons_right, icons_off };
 enum separator_color { SEP_FOREGROUND, SEP_AUTO, SEP_FRAME, SEP_CUSTOM };
 enum follow_mode { FOLLOW_NONE, FOLLOW_MOUSE, FOLLOW_KEYBOARD };
 enum markup_mode { MARKUP_NULL, MARKUP_NO, MARKUP_STRIP, MARKUP_FULL };
+enum mouse_action { do_action, close_current, push_all };
```

Comment:
Please match the enum value style with as above, all caps and prefixed with a relevant string e.g. `MOUSE_DO_ACTION`